command/delegate/delegate.go

```diff
@@ -477,7 +476,7 @@ func (c *delegateCommand) handleStep(w http.ResponseWriter, r *http.Request) {
 	logr = logr.
 		WithField("ip", inst.Address)
 
-	client, err := c.getLEClient(inst)
+	client, err := lehelper.GetLEClient(inst, c.runnerName)
```

Comment:
Drop one of the 'LE' you can tell from the package what type of client it is

Reply:
and vice versa

Reply:
changed.